src/main/clojure/jarl/eval.clj

```diff
@@ -314,7 +321,7 @@
   [source-index key-index value-index block-stmt state]
   (log/debugf "ScanStmt - scanning <%d>" source-index)
   (let [source (state/get-local state source-index)]
-    (if-not (coll? source) ; OPA IR docs states 'source' may not be an empty collection; but if we 'break' for such, statements like 'every x in [] { x != x }' will be 'undefined'.
+    (if-not (coll? source)                                  ; OPA IR docs states 'source' may not be an empty collection; but if we 'break' for such, statements like 'every x in [] { x != x }' will be 'undefined'.
```

Comment:
You really like text outside of your screen, do you? :)

Reply:
Not really. But the formatter does. Haven't fiddled with the default settings any.